### ISSUE TITLE
Consider flipping severity rating likert scale

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#2208]
The fact that some people expect the severity rating likert scale to be flipped (1 is least accessible, 5 is most accessible) comes up in most usability studies. This has been discussed quite a bit (including in #80 and especially #78).

It seems that there was discussion of this after usability studies in 2016, and we eventually decided to stick with the current rating system (the results of the user study is in #80 with the final decision being made in [this comment](https://github.com/ProjectSidewalk/SidewalkWebpage/issues/78#issuecomment-229526687) on that thread).

Given that we've decided on the current rating scheme (1 is most accessible, 5 is least accessible), I am not actually proposing that we change it here. What I'm hoping someone can fill in for me is _why_ we have settled on the scale we have. In #80 @nguyenist-zz describes some usability studies where people come across this issue, and @kotarohara says that they will discuss at the next meeting, and @kotarohara ultimately posts in that thread saying that we are sticking with the current design. But no one ever says _why_ we are sticking with the current design, nor gives any argument in favor of it.

So I'm just hoping we can give a rationale in this thread for why we chose the design, and then we can point people to this thread when it comes up.

@jonfroehlich do you have a rationale for why we ended up on the current design of the likert scale?

**Comment by jonfroehlich** [comment]
Thanks for this thoughtfully written post about a long-controversial issue. 

As your note references, we knew the scale order could be problematic. And, again, as you reference, we ran some informal usability tests that further surfaced this problem. If I recall correctly, we did not have strong data to convincingly suggest that there was a good, inherent order to the scale. We found, from our user studies, that maybe ~50% of users felt that the scale should be flipped but the other 50% did not. Then we took a vote as a team and decided to keep the same design (1 is most accessible, 5 is least accessible)—at the time, I should note that I believe I felt that 1 should be "worse" and 5 should be "best" (so, the opposite). But this was a long time ago.

In the end, I don't think there is a "right" answer here. The scale is not rating "quality." Instead, it is rating "severity." Thus, a 5 on a severity scale does indicate 'worst' and a 1 indicates 'best.'  

So, I suggest that we not change it now unless we have significant (new) user data showing us that it is a major problem. And I'm not sure we do currently.
